src/requirements/WalletActivity/components/AlchemyFirstTx.tsx

```diff
@@ -1,14 +1,40 @@
+import { Text } from "@chakra-ui/react"
+import { useWatch } from "react-hook-form"
 import { RequirementFormProps } from "requirements"
 import BlockNumberFormControl from "./BlockNumberFormControl"
 
-const AlchemyFirstTx = ({ baseFieldPath }: RequirementFormProps): JSX.Element => (
-  <BlockNumberFormControl
-    baseFieldPath={baseFieldPath}
-    dataFieldName="maxAmount"
-    label="Wallet created earlier than (date)"
-    isRequired
-    formHelperText="The date of the wallet's first transaction"
-  />
-)
+const AlchemyFirstTx = ({ baseFieldPath }: RequirementFormProps): JSX.Element => {
+  const [minAmount, maxAmount] = useWatch({
+    name: [
+      `${baseFieldPath}.data.timestamps.minAmount`,
+      `${baseFieldPath}.data.timestamps.maxAmount`,
+    ],
+  })
+
+  return (
+    <>
+      <Text colorScheme="gray" fontSize="sm" mt={-2} pl={3}>
```

Comment:
We actually already have a helper text inside the `BlockNumberFormControl` component, could you please check why it's not showing up? 👀